src/ShipPosition.js

```diff
@@ -0,0 +1,101 @@
+const Sequelize  = require('sequelize');
+const GameBd = require('.././database/GameModel.js')
+const PositionShipBd = require('.././database/PositionShip.js')
+const BoardBd = require('.././database/BoardModel.js')
+
+const valueMaxOfShips = 6
+
+class ShipPosition {
+  constructor({positionX = 0, positionY = 0, orientation = 'h', type = 1} = {}) {
+    this.positionX = positionX;
+    this.positionY = positionY;
+    this.orientation = orientation;
+    this.type = type;
+  }
+
+  static create({gameId = 0, playerId = 0, shipPositions = []} = {}) {
+    return GameBd.findOne({ where: 
+      Sequelize.or({ playerId1: playerId }, { playerId2: playerId })
+    })
+    .then((gameModel) => {
+      if(gameModel === null) {
+        return "The ships could not be positioned because the Player does not exist."
```

Comment:
this messages should be returned as errors to the user, (HTTP erros with the proper status code) so instead of returnin an string throw an error, catch it in the api layer (index.js file), build the error object and return that to the user:
```javascript
throw "The ships could not be positioned because the Player does not exist."

// in index.js
.catch(error => {
   console.error(error)
   res.status(SOME_STATUS_CODE_HERE).send({
       message: err
   })
})
```

Reply:
The messages were changed (throws) in the last commit